se2_planning/src/StateValidator.cpp

```diff
@@ -20,4 +20,15 @@ bool SE2stateValidator::isStateValid(const State& state) const {
   return true;
 }
 
+bool StateValidator::isLocked() const {
+  return isLocked_;
+}
+
+void StateValidator::lock() {
```

Comment:
Are these methods supposed to be thread safe? Or is it just here to prevent callbacks from updating the map?

Reply:
It is there to prevent a callback from updating the map when the map is used by the planner. Isn't this about thread safety?

Reply:
yes, you can leave them as they are.